stan/math/prim/prob/bernoulli_ccdf_log.hpp

```diff
@@ -2,7 +2,7 @@
 #define STAN_MATH_PRIM_SCAL_PROB_BERNOULLI_CCDF_LOG_HPP
```

Comment:
Header guards: find them all with `git grep --name-only PRIM_SCAL_PROB` (same for MAT).

Reply:
Thanks for catching that!